packages/service/ni_measurement_plugin_sdk_service/measurement/service.py

```diff
@@ -218,7 +218,16 @@ def __init__(
                 f"Service class '{service_class}' not found in '{service_config_file}'"
             )
 
+        if version:
+            warnings.warn(
+                "The 'version' parameter is deprecated. Specify the version in the .serviceconfig file instead.",
```

Comment:
Add a little more detail here? Something like `The 'version' constructor parameter is deprecated.  Specify the version using the 'version' field in the .serviceconfig file instead.`

Same in the message below: tell them which json field to add to the .serviceconfig.